react/components/molecules/input-animated/input-animated.js

```diff
@@ -0,0 +1,271 @@
+import React, { PureComponent } from "react";
+import { Animated, StyleSheet, View } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+import { Input, Touchable } from "../../atoms";
+
+export class InputAnimated extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            header: PropTypes.string,

```

Comment:
Call this `label`. I believe it's the term we generally use for this kind of stuff